Training25/Program.cs

```diff
@@ -3,11 +3,52 @@
 // Copyright (c) Metamation India.
 // ------------------------------------------------------------------
 // Program.cs
-// Program on main branch.
+// Program to check whether a password is strong or not.
 // ------------------------------------------------------------------------------------------------
+using System.Text;
+using static System.Console;
+
 namespace Training25;
+
 internal class Program {
-   static void Main (string[] args) {
-      Console.WriteLine ("Hello, World!");
+   static void Main () {
+      while (true) {
+         Write ("Enter a password: ");
+         string? pwd = ReadLine ();
+         if (string.IsNullOrEmpty (pwd)) {
+            Msg ("Enter a valid input\n");
+            continue;
+         }
+         if (TryParsePwd (pwd, out string errors)) {

```

Comment:
```suggestion
         if (TryParsePwd (pwd, out string error)) {
```
Misleading variable name